timemachine/fe/topology.py

```diff
@@ -214,7 +241,9 @@ def get_component_idxs(self) -> List[NDArray]:
         """
         return [np.arange(self.get_num_atoms())]
 
-    def parameterize_nonbonded(self, ff_q_params, ff_q_params_intra, ff_lj_params, lamb: float, intramol_params=True):
+    def parameterize_nonbonded(
+        self, ff_q_params, ff_q_params_intra, ff_q_params_solv, ff_lj_params, lamb: float, intramol_params=True
+    ):
```

Comment:
Per offline discussion, should assert that `ff_q_params_solv` is None to make it clear that user shoudn't pass this and it is only to maintain a consistent API.

Same with Line 479